src/component/dataZoom/SliderZoomView.ts

```diff
@@ -816,6 +817,7 @@ class SliderZoomView extends DataZoomView {
                         start: range[0], end: range[1]
                     }).valueWindow
                     : axisProxy.getDataValueWindow();
+                this._valueRange = dataInterval;
```

Comment:
So if `showDetail` is false, this value range is not updated. Is this expected?

Reply:
No, I will rearrage it